### PR TITLE
Tickets/preops 5100

### PR DESCRIPTION
**Pull request by Madejski**
This is a pull request for Yumi to review the revised DP0.3 Portal tutorial 4 (phase curves).  The tutorial reflects the new UI, but does not have any substantive changes.  Let me know once you reviewed it - but please don't merge yet.

### CODE REVIEW

File: _static/portal_tut04_step02c.png
Comment (galaxyumi) [inline _static/portal_tut04_step02c.png:1]: I can't see the Y column and Error on Y-axis in the screenshot.

**Review by galaxyumi — DISMISSED** [review]
I left the only comment about the screenshot in S2.4. Other than that, it looks good to me. Nice job!

File: _static/portal_tut04_step02c.png
Comment (Madejski) [reply _static/portal_tut04_step02c.png:1]: Thanks!  I will fix it.

Comment (Madejski) [reply _static/portal_tut04_step02c.png:1]: I just fixed it, and also cleaned up labels on the two plots in that part.  Thanks!

**Review by ChristinaAdair — APPROVED** [review]

**Pull request merged**